pkg/api/controllers/metrics.go

```diff
@@ -280,3 +280,27 @@ func (m *MetricsPortal) CollectMetrics() {
 
 	return
 }
+
+func (m *MetricsPortal) GetUrls() {
+	if !policy.Authorize(m.Ctx, "metrics:urls") {
+		return
```

Comment:
The error info?

Reply:
Checked, in case the authorization fails, we return the status code **403 Forbidden** with the message 
```
{"code":403,"message":"operation is not permitted"}
```